jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1155,6 +1155,10 @@ public Method getDeclaredMethod(String name, Class<?>... parameterTypes) throws
  */
 private native Method[] getDeclaredMethodsImpl();
 
+
+private transient Class<?> cachedDeclaringClass;
```

Comment:
Please move all the field declarations to the top of the file with the other field declarations.